### ISSUE TITLE
Compatibility With Debian 8

### ISSUE THREAD

**Issue by aviakasurvivor** [GoogleCloudPlatform/compute-video-demo-salt#7]
Hi Team,

I am not able to work this out with Debian 8 in GCP as Debian 7 is deprecated.
This Salt Minions are getting created but not running.

Regards,
Avi

**Comment by laundry-96** [comment]
I know this is a year late, but I am running with Debian 9, and this issue is happening to me too!

**Comment by laundry-96** [comment]
After a wee bit of digging into the created salt-minion, and listing the status of salt-minion, we get this this:
`● salt-minion.service - The Salt Minion
   Loaded: loaded (/lib/systemd/system/salt-minion.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Sat 2019-02-02 03:37:14 UTC; 3min 15s ago
     Docs: man:salt-minion(1)
           file:///usr/share/doc/salt/html/contents.html
           https://docs.saltstack.com/en/latest/contents.html
  Process: 3334 ExecStart=/usr/bin/salt-minion (code=exited, status=1/FAILURE)
 Main PID: 3334 (code=exited, status=1/FAILURE)

Feb 02 03:37:13 minecraft-hexxit systemd[1]: Starting The Salt Minion...
Feb 02 03:37:13 minecraft-hexxit systemd[1]: Started The Salt Minion.
Feb 02 03:37:14 minecraft-hexxit salt-minion[3334]: [INFO    ] Setting up the Salt Minion "minecraft-hexxit"
Feb 02 03:37:14 minecraft-hexxit salt-minion[3334]: [INFO    ] Starting up the Salt Minion
Feb 02 03:37:14 minecraft-hexxit salt-minion[3334]: [INFO    ] Starting pull socket on /var/run/salt/minion/minion_event_97c29e6dd1_pull.ipc
Feb 02 03:37:14 minecraft-hexxit salt-minion[3334]: [INFO    ] Creating minion process manager
Feb 02 03:37:14 minecraft-hexxit salt-minion[3334]: [ERROR   ] Invalid keyword 'standard' for variable 'master_type'
Feb 02 03:37:14 minecraft-hexxit systemd[1]: salt-minion.service: Main process exited, code=exited, status=1/FAILURE
Feb 02 03:37:14 minecraft-hexxit systemd[1]: salt-minion.service: Unit entered failed state.
Feb 02 03:37:14 minecraft-hexxit systemd[1]: salt-minion.service: Failed with result 'exit-code'.`

**Comment by laundry-96** [comment]
So, my guess, is that we need to somehow change master_type to a different value?

**Comment by laundry-96** [comment]
Yes, updating master_type to 'str' in /etc/salt/cloud.profiles fixes this problem. I will make a PR.
